lib/pure/strutils.nim

```diff
@@ -1865,7 +1872,7 @@ func find*(s: string, sub: char, start: Natural = 0, last = 0): int {.rtl,
   ##
   ## Searching is case-sensitive. If `sub` is not in `s`, -1 is returned.
   ## Otherwise the index returned is relative to `s[0]`, not `start`.
-  ## Use `s[start..last].rfind` for a `start`-origin index.
+  ## Subtract `start` from the result for a `start`-origin index.
```

Comment:
```suggestion
  ## Subtracts `start` from the result for a `start`-origin index.
```

Reply:
@xflywind your suggestions in this case are wrong - "Subtract" here is a call for action, not the description of the procedure itself. It basically tells: "User, subtract `start` from the result to get a `start`-origin index"

Same applies to your suggestion below

Reply:
I see, thanks